<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KFindTransactionDlg</class>
 <widget class="QDialog" name="KFindTransactionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search transactions</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="m_tabWidget">
     <widget class="QWidget" name="m_resultPage">
      <attribute name="title">
       <string>Result</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="KMyMoneyRegister::Register" name="m_register" native="true">
         <property name="numRows" stdset="0">
          <number>0</number>
         </property>
         <property name="numCols" stdset="0">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_foundText">
         <property name="text">
          <string>F</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyRegister::Register</class>
   <extends>QWidget</extends>
   <header>register.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ButtonGroup1"/>
  <buttongroup name="ButtonGroup2"/>
 </buttongroups>
</ui>
